tests/tools/polygon_tools_test.py

```diff
@@ -1,99 +1,6 @@
 from wzdx.tools import polygon_tools
```

Comment:
We're only covering two of the methods in the polygon_tools.py file here, should we explicitly cover the rest?

Reply:
I did all of the testing for these methods visually. it is kinda tough to add meaningful unit tests, short of just hardcoding the output and verifying that it matches, do you have any ideas?

Reply:
I added unit tests for all of the methods